components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -93,52 +125,121 @@ export default function () {
     if (!!workspaceCreationResult) {
       setWorkspaceCreationResult(undefined);
     }
     try {
+      setPrebuildWasTriggered(true);
       await getGitpodService().server.setProjectConfiguration(project.id, gitpodYml);
       const result = await getGitpodService().server.createWorkspace({
         contextUrl: `prebuild/${project.cloneUrl}`,
         mode: CreateWorkspaceMode.ForceNew,
       });
       setWorkspaceCreationResult(result);
     } catch (error) {
-      setEditorError(<span>{String(error)}</span>);
+      setEditorMessage(<EditorMessage type="warning" heading="Could not run prebuild." message={String(error)}/>);
     }
   }
 
+  const onInstanceUpdate = (instance: WorkspaceInstance) => {
+    setPrebuildPhase(instance.status.phase);
+  }
+
   useEffect(() => { document.title = 'Configure Project — Gitpod' }, []);
 
-  return <div className="flex flex-col mt-24 mx-auto items-center">
-    <h1>Configure Project</h1>
-    <p className="text-gray-500 text-center text-base">Fully-automate your project's dev setup. <a className="gp-link" href="https://www.gitpod.io/docs/references/gitpod-yml">Learn more</a></p>
-    <div className="mt-4 w-full flex">
-      <div className="flex-1 m-8">
-        {editorError && <AlertBox className="mb-2">{editorError}</AlertBox>}
-        {!isEditorDisabled && <select className="w-full" defaultValue="" onChange={e => setGitpodYml(e.target.value)}>
-          <option value="" disabled={true}>…</option>
-          {Object.entries(TASKS).map(([ name, value ]) => <option value={value}>{name}</option>)}
-        </select>}
-        {!!dockerfile && <div className="flex justify-center border-b border-gray-200 dark:border-gray-800">
+  return <>
+    <Header title="Configuration" subtitle="View and edit project configuration." />
+    <div className="lg:px-28 px-10 mt-8 flex space-x-4">
+      <div className="w-96 h-96 rounded-xl overflow-hidden relative bg-gray-100 dark:bg-gray-700 flex flex-col">
+        <div className="flex bg-gray-50 dark:bg-gray-800 border-b border-gray-200 dark:border-gray-600 px-6 pt-3">
           <TabMenuItem name=".gitpod.yml" selected={selectedEditor === '.gitpod.yml'} onClick={() => setSelectedEditor('.gitpod.yml')} />
-          <TabMenuItem name=".gitpod.Dockerfile" selected={selectedEditor === '.gitpod.Dockerfile'} onClick={() => setSelectedEditor('.gitpod.Dockerfile')} />
-        </div>}
+          {!!dockerfile && <TabMenuItem name=".gitpod.Dockerfile" selected={selectedEditor === '.gitpod.Dockerfile'} onClick={() => setSelectedEditor('.gitpod.Dockerfile')} />}
+        </div>
+        {editorMessage}
         <Suspense fallback={<div />}>
           {selectedEditor === '.gitpod.yml' &&
-            <MonacoEditor classes="mt-4 w-full h-64" disabled={isEditorDisabled} language="yaml" value={gitpodYml} onChange={setGitpodYml} />}
+            <MonacoEditor classes="w-full flex-grow" disabled={isEditorDisabled} language="yaml" value={gitpodYml} onChange={setGitpodYml} />}
```

Comment:
@gtsiolis Sure! But which gray? The [specs](https://www.figma.com/file/iYeLdIMjMdSpTgFGEAicmk/Project-Configuration-%234648?node-id=0%3A1) have a white background in every non-disabled editor.

Reply:
> The specs have a white background in every non-disabled editor.

That's correct!

> But which gray?

@jankeromnes What do you think of using `gray-400` for the text and `gray-100` for the background when the editor is disabled? See [relevant frame in specs](https://www.figma.com/file/iYeLdIMjMdSpTgFGEAicmk/Project-Configuration-4648?node-id=1417%3A1781).

Reply:
> That's correct!

Aha, "when the configuration is non-editable" was common to both "a)" and "b)" 😅 (sorry, I understood "a) (always) and b) (when disabled)")

> What do you think of using `gray-400` for the text and `gray-100` for the background when the editor is disabled?

Sure! But won't that be problematic in terms of minimum acceptable contrast? (I know for me it might cause some eye strain.) Happy to try the suggested colors as a first iteration!

Reply:
> But won't that be problematic in terms of minimum acceptable contrast?

You're right! Thanks for caring about accessibility. What do you think of chaning the background to `gray-100` but keeping the text color as is (`gray-600`) as this combination passes `WCAG AA` color checks?

Reply:
Sounds good to me, thanks! Left the text color as is, but adjusted the background according to whether editing is disabled or not (in both light and dark themes). Result looks great 😁 ✨